### PR TITLE
Update rasterize algo with QGIS 2.14

### PR DESCRIPTION
**Pull request by Gustry**

### CODE REVIEW

File: safe/impact_function/test/test_impact_function.py [hunk 1]
```diff
@@ -384,9 +386,12 @@ def test_scenarii(self):
             'raster_continuous_on_raster_population': False,
         }
 
+        # If we want to invert the selection.
+        invert = False
+
         path = standard_data_path('scenario')
         for scenario, enabled in scenarii.iteritems():
```
Comment (timlinux) [inline safe/impact_function/test/test_impact_function.py]: BTW I think scenarios is a more established plural for scenario that scenarii ....

**Comment by timlinux** [comment]
LGTM when tests pass

**Comment by Charlotte-Morgan** [comment]
hi @Gustry - I get #3719 assertion error still with this PR

File: safe/impact_function/test/test_impact_function.py [hunk 2]
```diff
@@ -384,9 +386,12 @@ def test_scenarii(self):
             'raster_continuous_on_raster_population': False,
         }
 
+        # If we want to invert the selection.
+        invert = True
+
         path = standard_data_path('scenario')
         for scenario, enabled in scenarii.iteritems():
```
Comment (timlinux) [inline safe/impact_function/test/test_impact_function.py]: scenarii -> scenarios

Comment (lucernae) [inline safe/impact_function/test/test_impact_function.py]: Ha, I thought it was intended. It was like latin or something :)

Comment (ismailsunni) [inline safe/impact_function/test/test_impact_function.py]: Hah, I thought scenarii is plural also. It should be scenari anyway in Italian. Reference http://english.stackexchange.com/questions/11775/what-is-the-plural-of-scenario

Comment (lucernae) [inline safe/impact_function/test/test_impact_function.py]: maybe we catch up some latin or italian while in rome, @ismailsunni

**Pull request merged**